tests/test_claude_wrapper_hooks.py

```
@@ -1410,6 +1410,45 @@ def test_live_socket_preserves_claude_auth_for_resume_launch(failures: list[str]
+        expected: dict[str, str] = {}
+
+        def setup_env(tmp: Path) -> dict[str, str]:
+            home = tmp / "home"
+            default_root = home / ".claude"
+            (default_root / "projects" / "-work").mkdir(parents=True)
+            (default_root / "projects" / "-work" / f"{session_id}.jsonl").write_text(
+                "{}\n", encoding="utf-8"
+            )
+            selected_root = home / ".subrouter" / "codex" / "claude" / "aziz-claude-1"
+            (selected_root / "projects").mkdir(parents=True)
+            expected["path"] = str(selected_root)
```

Ruff (0.15.20)

[warning] Function definition does not bind loop variable `expected`

(B023)